dashboard/app/assets/javascripts/small_footer.js.jsx

```diff
@@ -57,22 +57,21 @@ window.dashboard.footer = (function () {
       getInitialState: function () {
         return {
           menuState: MenuState.MINIMIZED,
-          copyrightStyle: null,
-          moreMenuStyle: null
+          baseWidth: 0,
+          baseHeight: 0
         };
       },
 
       componentDidMount: function () {
-        var originalSetState = this.setState;
+        this.captureBaseElementDimensions();
+        window.addEventListener('resize', this.captureBaseElementDimensions);
```

Comment:
@Bjvanminnen this is a fix for a regression that I _think_ happened when this was refactored to React.  Here's the bad "before" behavior:

On page load we'd get the correct dimensions of the footer and position the menu correctly:
![screenshot from 2015-12-03 16-05-55](https://cloud.githubusercontent.com/assets/1615761/11578178/e86bc9d4-99d7-11e5-90ee-2f8f37766a17.png)

But the footer resizes with the left column (whether by dragging the grippy or doing a window resize) and the footer base dimensions weren't being updated in the React component, causing us to lose the bottom menu item and have an incorrect menu width:
![screenshot from 2015-12-03 16-06-09](https://cloud.githubusercontent.com/assets/1615761/11578193/00354752-99d8-11e5-9308-c090532c4dc7.png)

By updating on the window resize event (which we also fire when dragging the grippy) we correctly update the menu width/position too:
![screenshot from 2015-12-03 16-08-45](https://cloud.githubusercontent.com/assets/1615761/11578221/3edb1f9a-99d8-11e5-8049-0aa93784e6a1.png)